hubble/cmd/common/conn/conn.go

```diff
@@ -66,12 +49,9 @@ func Init(vp *viper.Viper) error {
 }
 
 // New creates a new gRPC client connection to the target.
-func New(ctx context.Context, target string, dialTimeout time.Duration) (*grpc.ClientConn, error) {
-	dialCtx, cancel := context.WithTimeout(ctx, dialTimeout)
-	defer cancel()
-
+func New(target string) (*grpc.ClientConn, error) {
 	t := strings.TrimPrefix(target, defaults.TargetTLSPrefix)
-	conn, err := grpc.DialContext(dialCtx, t, grpcDialOptions...)
+	conn, err := grpc.NewClient(t, grpcDialOptions...)
 	if err != nil {
 		return nil, fmt.Errorf("failed to connect to '%s': %w", target, err)
```

Comment:
nitpicking: after this patch, the error message is slightly misleading. From https://pkg.go.dev/google.golang.org/grpc#NewClient (emphasis mine)
> NewClient creates a new gRPC "channel" for the target URI provided. _No I/O is performed_.
```suggestion
		return nil, fmt.Errorf("failed to create gRPC client to '%s': %w", target, err)
```